src/librustc_target/spec/windows_gnu_base.rs

```diff
@@ -66,8 +66,7 @@ pub fn opts() -> TargetOptions {
     );
 
     TargetOptions {
-        // FIXME(#13846) this should be enabled for windows
-        function_sections: false,
+        function_sections: true,
```

Comment:
The line can be just removed because `true` is the default.

Reply:
Right, thanks.